### PR TITLE
Add Tokopedia

### LINKED ISSUE #3695: Please add tokopedia.com
Tokopedia is one of the popular online marketplaces in Indonesia and is the only one that supports 2FA via Google Authenticator. Users can enable it in the Account Settings menu.

https://www.tokopedia.com

### PR DESCRIPTION
**Pull request by dsoegijono**
The doc is in Indonesian, but it's the only page I could find. It only shows how to enable Google Authenticator, but the screenshot in #3695 shows SMS and phone options too.

### CODE REVIEW

File: _data/retail.yml [hunk 1]
```diff
@@ -423,4 +432,3 @@ websites:
       tfa: No
       twitter: zooplusDE
       facebook: zooplusde
-
```
Comment (mxxcon) [inline _data/retail.yml:426]: we do want trailing white space 😉 see https://github.com/2factorauth/twofactorauth/blob/master/.editorconfig#L10

**Review by mxxcon — DISMISSED** [review]

File: _data/retail.yml (hunk 1 above)
Comment (dsoegijono) [reply _data/retail.yml:426]: Looks like there's already one, there was an extra one. GitHub would show [this icon](https://0.readthedocs.io/en/latest/_images/github-diff-no-newline-warning.png) if there wasn't one.

**Review by Carlgo11 — APPROVED** [review]

**Review by jtagcat — CHANGES_REQUESTED** [review]
png a bit big

**Review by jtagcat — APPROVED** [review]

**Comment by kmpoppe** [comment]
Closes #3695

**Review by kmpoppe — APPROVED** [review]
Image size could only be reduced by a tiny amount, as the logo is relatively complex in colour.

**Pull request merged**